v2/charts/azure-service-operator/templates/apps_v1_deployment_azureserviceoperator-controller-manager.yaml

```
@@ -177,22 +181,23 @@ spec:
         - mountPath: /var/run/secrets/tokens
           name: azure-identity
           readOnly: true
-   {{- if or (eq .Values.multitenant.enable false) (eq .Values.azureOperatorMode "webhooks") }}
+    {{- if or (eq .Values.multitenant.enable false) (eq .Values.azureOperatorMode "webhooks") }}
```

What are you lining these indentations up with?

I'd be fine with all left justified:
```suggestion
{{- if or (eq .Values.multitenant.enable false) (eq .Values.azureOperatorMode "webhooks") }}
```

or justified at the same identation as what it's wrapping:
```suggestion
        {{- if or (eq .Values.multitenant.enable false) (eq .Values.azureOperatorMode "webhooks") }}
```

Not sure I follow the pattern you followed here though?